external/webclient/webclient.c

```diff
@@ -939,10 +940,12 @@ static pthread_addr_t wget_base(void *arg)
 	struct http_client_request_t *param = (struct http_client_request_t *)arg;
 	struct http_client_response_t response = {0, };
 	int read_finish = false;
+	int send_len = 0;
 
```

Comment:
you should add null check for arg first.

Reply:
This is existing code